src/rich-text/commands.ts

```diff
@@ -652,23 +654,15 @@ export const createMenu = (options: CommonViewOptions): Plugin =>
                 {
                     key: "insertImage",
                     command: insertImageCommand,
-                    dom: makeMenuIcon(
-                        "Image",
-                        "Image",
-                        "insert-image-btn"
-                    ),
+                    dom: makeMenuIcon("Image", "Image", "insert-image-btn"),
```

Comment:
I have no idea why this line reformatted. It formats exactly the same locally as you have here, so... I'm calling it good.

No changes necessary.